atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -52,6 +52,7 @@ public SerializableTransactionManager delegate() {
                     // TODO(ssouza): replace with KVS healthcheck status when that gets implemented.
                     manager.getKeyValueService().getClusterAvailabilityStatus();
                 } catch (NotInitializedException e) {
+                    log.warn("The KeyValueService is not initialized yet!");
```

Comment:
Similar to comments on the other PR, this sounds like an expected situation, so should probably only be `info`.